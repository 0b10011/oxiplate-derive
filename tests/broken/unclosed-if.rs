use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = "{%- if should_do -%}{{ action }}"]
struct Data {
    should_do: bool,
    action: &'static str,
}

fn main() {
    let data = Data {
        should_do: true,
        action: "do something",
    };

    print!("{}", data);
}
