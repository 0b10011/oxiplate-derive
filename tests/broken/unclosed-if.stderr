error: "if" statement is never closed (unexpected end of template)
 --> tests/broken/unclosed-if.rs:4:54
  |
4 | #[oxiplate_inline = "{%- if should_do -%}{{ action }}"]
  |                                                      ^
